### PR TITLE
[#385] [Fastlane Swift] - Update lanes from Swift for GitHub Action's workflow and bitrise.yml

### PR DESCRIPTION
**Pull request by nkhanh44**
#385 

## What happened

- Updated the lanes to Swift for GitHub Action's workflow and bitrise.yml.
 
## Insight

- Updated from The Ruby lane `build_and_test` in [Fastfile](https://github.com/nimblehq/ios-templates/blob/main/fastlane/Fastfile) to the Swift lane `buildAndTestLane` in [Fastfile.swift](https://github.com/nimblehq/ios-templates/blob/migrate-develop/fastlane/Fastfile.swift). Similar for other lanes in File changed. 
 
## Proof Of Work
I created another repository to trigger the lanes as the following sections:
### GitHub Action

- Build and Test:
<details>
<summary>buildAndTestLane</summary>
<be>

</details>

- Clean up previous code coverage report:
<details>
<summary>cleanUpOutputLane</summary>
<br>

</details>

- Match Ad-hoc:
<details>
<summary>syncAdHocStagingCodeSigningLane</summary>
<br>

</details>

<details>
<summary> syncAdHocProductionCodeSigningLane</summary>
<br>

</details>

<details>
<summary>syncAppStoreCodeSigningLane</summary>
<br>

</details>

- Build App and Distribute:

<details>
<summary>buildStagingAndUploadToFirebaseLane</summary>
<br>

</details>

<details>
<summary>buildProductionAndUploadToFirebaseLane</summary>
<br>

</details>

<details>
<summary>buildAndUploadToAppStoreLane</summary>
<br>

</details>

### Bitrise

- Build and Test:
<details>
<summary>buildAndTestLane</summary>
<be>

</details>

- Clean up output:
<details>
<summary>cleanUpOutputLane</summary>
<br>

</details>

- Build App and Distribute:

<details>
<summary>buildStagingAndUploadToFirebaseLane</summary>
<br>

</details>

<details>
<summary>buildProductionAndUploadToFirebaseLane</summary>
<br>

</details>

<details>
<summary>buildAndUploadToAppStoreLane</summary>
<br>

</details>

### CODE REVIEW

File: .github/workflows/automatic_pull_request_review.yml [hunk 1]
```diff
@@ -42,12 +42,12 @@ jobs:
       run: bundle exec pod install
 
     - name: Build and Test
-      run: bundle exec fastlane build_and_test
+      run: bundle exec fastlane buildAndTestLane --disable_runner_upgrades
```
Comment (vnntsu) [inline .github/workflows/automatic_pull_request_review.yml:45]: @nkhanh44 FYI, we don't commit `FastlaneRunner` executable file to git, so we don't need to disable its upgrade. On CI, it works well without `--disable_runner_upgrades`

Comment (nkhanh44) [reply .github/workflows/automatic_pull_request_review.yml]: Thank you for the information, I updated it at https://github.com/nimblehq/ios-templates/pull/395/commits/3e288e5aa62e42dd31a650742d9d230a168ae9f4 😄.

File: .github/workflows/deploy_AppStore.yml [hunk 2]
```diff
@@ -62,14 +62,14 @@ jobs:
...
-      run: bundle exec fastlane sync_appstore_signing
+      run: bundle exec fastlane syncAppStoreCodeSigningLane
       env:
         MATCH_PASSWORD: ${{ secrets.MATCH_PASS }}
 
     - name: Build App and Distribute to AppStore
-      run: bundle exec fastlane build_and_upload_appstore_app
+      run: bundle exec fastlane buildAndUploadToAppStoreLane
       env:
         APPSTORE_CONNECT_API_KEY: ${{ secrets.APPSTORE_CONNECT_API_KEY }}
```
Comment (blyscuit) [inline .github/workflows/deploy_AppStore.yml:75]: Let's fix these last line break too 🙌

File: .github/workflows/deploy_Firebase.yml [hunk 3]
```diff
@@ -67,14 +67,14 @@ jobs:
...
-      run: bundle exec fastlane sync_adhoc_signing
+      run: bundle exec fastlane syncAdHocStagingCodeSigningLane
       env:
         MATCH_PASSWORD: ${{ secrets.MATCH_PASS }}
 
     - name: Build App and Distribute to Firebase
-      run: bundle exec fastlane build_and_upload_staging_app
+      run: bundle exec fastlane buildStagingAndUploadToFirebaseLane
       env:
         FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
```
Comment (blyscuit) [inline .github/workflows/deploy_Firebase.yml:80]: 🙌

File: .github/workflows/deploy_Release_Firebase.yml [hunk 4]
```diff
@@ -61,14 +61,14 @@ jobs:
...
-      run: bundle exec fastlane sync_adhoc_production_signing
+      run: bundle exec fastlane syncAdHocProductionCodeSigningLane
       env:
         MATCH_PASSWORD: ${{ secrets.MATCH_PASS }}
 
     - name: Build Production App and Distribute to Firebase
-      run: bundle exec fastlane build_and_upload_production_app
+      run: bundle exec fastlane buildProductionAndUploadToFirebaseLane
       env:
         FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
```
Comment (blyscuit) [inline .github/workflows/deploy_Release_Firebase.yml:74]: 🙌

File: .github/workflows/deploy_AppStore.yml (hunk 2 above)
Comment (nkhanh44) [reply .github/workflows/deploy_AppStore.yml]: Nice catch! I fixed all at https://github.com/nimblehq/ios-templates/pull/395/commits/934a26c95c92af49deb09db9517aedfb2e151176 😄.

**Review by vnntsu — COMMENTED** [review]
@nkhanh44 Please make update for `test_install_script.yml` and `test_upload_build_to_test_flight.yml`. And now, we can uncomment these 2 files.

File: .github/workflows/automatic_pull_request_review.yml [hunk 5]
```diff
@@ -42,12 +42,12 @@ jobs:
       run: bundle exec pod install
 
     - name: Build and Test
-      run: bundle exec fastlane build_and_test
+      run: bundle exec fastlane buildAndTestLane
```
Comment (suho) [inline .github/workflows/automatic_pull_request_review.yml:45]: I think we can remove the suffix `Lane`. You can try to run `bundle exec fastlane buildAndTest`. If it's ok, please remove all suffixes :D

**Review by blyscuit — APPROVED** [review]

File: .github/workflows/automatic_pull_request_review.yml (hunk 5 above)
Comment (nkhanh44) [reply .github/workflows/automatic_pull_request_review.yml:45]: `bundle exec fastlane buildAndTest`
It works. Thank you 🙌 .

Comment (suho) [reply .github/workflows/automatic_pull_request_review.yml:45]: @nkhanh44 It's time to update this PR :D

Comment (nkhanh44) [reply .github/workflows/automatic_pull_request_review.yml]: I fixed it at https://github.com/nimblehq/ios-templates/pull/395/commits/f92a92b9406367940c03752b37d76935337fe6e7 :D.

**Comment by nkhanh44** [comment]
@vnntsu I made update for `test_install_script.yml` and `test_upload_build_to_test_flight.yml` at https://github.com/nimblehq/ios-templates/pull/395/commits/16c56716c979cb6fc9cac61051716a787045fc31. Please help me to check it again :D. Thank you 📿 .

=> `test_install_script.yml` works at [this job](https://github.com/nimblehq/ios-templates/actions/runs/3780206918).
=> `test_upload_build_to_test_flight.yml` works at [this job](https://github.com/nimblehq/ios-templates/actions/runs/3780207026/jobs/6426099164).

**Comment by vnntsu** [comment]
> @vnntsu I made update for `test_install_script.yml` and `test_upload_build_to_test_flight.yml` at [16c5671](https://github.com/nimblehq/ios-templates/commit/16c56716c979cb6fc9cac61051716a787045fc31). Please help me to check it again :D. Thank you 📿 .
> 
> => `test_install_script.yml` works at [this job](https://github.com/nimblehq/ios-templates/actions/runs/3780206918). => `test_upload_build_to_test_flight.yml` works at [this job](https://github.com/nimblehq/ios-templates/actions/runs/3780207026/jobs/6426099164).

@nkhanh44 Well done <3 The changes look good to me now. Please also check the CI with a failed step.

**Review by vnntsu — APPROVED** [review]

**Review by phongvhd93 — APPROVED** [review]

**Pull request merged**